apps/catalog/catalog/src/app/dashboard/dashboard.component.scss

```
@@ -1,4 +1,21 @@
-﻿:host {
-  display: block;
-  height: 100vh;
-}
+﻿@import "@angular/flex-layout/mq";
+
+layout-dashboard aside {
+  mat-nav-list {
+    display: flex;
+    flex-direction: column;
+  }
+
+  @include layout-bp(gt-xs) {
+    &:not(.opened) auth-sidenav,
+    &.opened auth-sidenav {
+      display: none;
+    }
```

You cover all the case, no need for ".opened": 
```suggestion
    auth-sidenav {
      display: none;
    }
```

---

Yeah I know but if I don't do this `display: none` is overwritten by another property of Angular who have priority on it